src/appshell/internal/appshellconfiguration.cpp

```diff
@@ -188,6 +188,17 @@ void AppShellConfiguration::setNeedShowSplashScreen(bool show)
     settings()->setSharedValue(SPLASH_SCREEN_VISIBLE_KEY, Val(show));
 }
 
+QString AppShellConfiguration::preferencesDialogLastOpenedPageId() const
+{
+    return (m_preferencesDialogCurrentPageIdQString);
+}
+
+void AppShellConfiguration::setPreferencesDialogLastOpenedPageId(QString lastOpenedPageId)
+{
+    LOGI() << "tpacebes nuevo lastOpenedPageId" << lastOpenedPageId;
```

Comment:
Please remove this before we merge the PR; if all temporary debug log messages would be kept, then the logs would become a bit too cluttered :)